src/draw/lv_draw_gradient.c

```diff
@@ -0,0 +1,316 @@
+/**
+ * @file lv_draw_gradient.c
+ *
+ */
+
+/*********************
```

Comment:
I've noticed that this file should be also in the `draw/sw` folder

Reply:
Moved.